src/Protocol/DFRN.php

```diff
@@ -465,7 +471,7 @@ public static function relocate($owner, $uid)
 		/* get site pubkey. this could be a new installation with no site keys*/
 		$pubkey = Config::get('system', 'site_pubkey');
 		if (! $pubkey) {
-			$res = Crypto::newKeypair(1024);
+			$res = FriendicaCrypto::newKeypair(1024);
```

Comment:
I'm no expert here, is 1024 good enough?

Reply:
In general, no, [RSA-1024 can be broken](https://www.techworld.com/news/security/rsa-1024-bit-private-key-encryption-cracked-3214360/), but for our use where, after being almost securely transmitted, we store plaintext data on cheap hosting boxes for years, it should be enough.

Reply:
Turns out the users' RSA keys already are initialized with 4096 bits! This is only for the server, I'll leave it be.